paper/src/main/java/com/denizenscript/denizen/paper/PaperModule.java

```diff
@@ -51,40 +51,44 @@ public static void init() {
             ScriptEvent.registerScriptEvent(PlayerChangesFramedItemScriptEvent.class);
         }
         ScriptEvent.registerScriptEvent(PlayerChoosesArrowScriptEvent.class);
+        ScriptEvent.registerScriptEvent(PlayerChunkUnloadScriptEvent.class);

```

Comment:
seems to exist in 1.17, with the only change since being the "NotNull" removal in 1.21. https://jd.papermc.io/paper/1.17/io/papermc/paper/event/packet/PlayerChunkUnloadEvent.html